kuma/wiki/apps.py

```diff
@@ -55,7 +14,9 @@ class WikiConfig(AppConfig):
     verbose_name = _("Wiki")
 
     def ready(self):
-        super(WikiConfig, self).ready()
+        '''Configure kuma.wiki after models are loaded.'''
+        # Register signal handlers
+        from . import signal_handlers  # noqa
 
         # Configure Elasticsearch connections for connection pooling.
         es_connections.configure(
```

Comment:
Unrelated to this PR, but I was just thinking that maybe this belongs under `search.apps.SearchConfig.ready`?

Reply:
I like that idea. Trying it out...